snuba/snuba_migrations/functions/0003_add_new_columns_to_raw_functions.py

```diff
@@ -0,0 +1,86 @@
+from dataclasses import dataclass
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, DateTime64, String
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget
+
+
+@dataclass(frozen=True)
+class NewColumn:
+    column: Column[Modifiers]
+    after: str | None
```

Comment:
Why would `after` ever be `None`?

Reply:
If there are columns for which we don't need to specify it.

The `after` attribute of the `AddColumn` class after all has it as an optional type (`Optional[str]`) 
https://github.com/getsentry/snuba/blob/master/snuba/migrations/operations.py#L286-L291